data/mods/ssb/moves.ts

```diff
@@ -211,7 +211,27 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 		target: "normal",
 		type: "Fire",
 	},
-
+	// Chloe
+	viosyni: {
```

Comment:
```suggestion
	vsni: {
```
afaik unicode is a bit tricky so leave out any non-english characters.